src/Pages/form/Components/ChiefComplaintSection.tsx

```diff
@@ -101,58 +76,204 @@ const ChiefComplaintSection: React.FC = () => {
+            control={
+              <PinkCheckbox
+                onChange={handleChange}
+                name="FacialWeakness"
+                id="FacialWeakness"
+              />
+            }
+            label="Facial weakness"
+          />
+          {symptomsCheckbox.FacialWeakness && (
+            <FormGroup row id="SubFacialWeakness" className={classes.checkbox}>
+              <FormControlLabel
+                control={
+                  <PinkCheckbox
+                    onChange={handleChange}
+                    name="LeftFacialWeakness"
+                  />
+                }
+                label="Left"
+              />
+              <FormControlLabel
+                control={
+                  <PinkCheckbox
+                    onChange={handleChange}
+                    name="RightFacialWeakness"
+                  />
+                }
+                label="Right"
+              />
+            </FormGroup>
+          )}
+          <FormControlLabel
+            control={
+              <PinkCheckbox
+                onChange={handleChange}
+                name="Hemiparesis"
+                id="Hemiparesis"
+              />
+            }
+            label="Hemiparesis"
+          />
+          {symptomsCheckbox.Hemiparesis && (
+            <FormGroup row id="SubHemiparesis" className={classes.checkbox}>
+              <FormControlLabel
+                control={
+                  <PinkCheckbox
+                    onChange={handleChange}
+                    name="LeftHemiparesis"
+                  />
+                }
+                label="Left"
+              />
+              <FormControlLabel
+                control={
+                  <PinkCheckbox
+                    onChange={handleChange}
+                    name="RightHemiparesis"
+                  />
+                }
+                label="Right"
+              />
+            </FormGroup>
+          )}
+          <FormControlLabel
+            control={
+              <PinkCheckbox
+                onChange={handleChange}
+                name="Hemiparesthesia"
+                id="Hemiparesthesia"
+              />
+            }
+            label="Hemiparesthesia"
+          />
+          {symptomsCheckbox.Hemiparesthesia && (
+            <FormGroup row id="SubHemiparesthesia" className={classes.checkbox}>
+              <FormControlLabel
+                control={
+                  <PinkCheckbox
+                    onChange={handleChange}
+                    name="LeftHemiparesthesia"
+                  />
+                }
+                label="Left"
+              />
+              <FormControlLabel
+                control={
+                  <PinkCheckbox
+                    onChange={handleChange}
+                    name="RightHemiparesthesia"
+                  />
+                }
+                label="Right"
+              />
+            </FormGroup>
+          )}
+          <FormControlLabel
```

Comment:
Actually, you can group all `<FormContralLabel />`. It will decrease your line of code.